src/main/java/core/basesyntax/MatrixOperation.java

```diff
@@ -5,19 +5,43 @@
  * максимального та мінімального елементів.
  */
 public class MatrixOperation {
-    public int[] getDiagonal(int[][] matrix) {
-        return null;
+    public int [] getDiagonal(int[][] matrix) {
```

Comment:
```suggestion
    public int[] getDiagonal(int[][] matrix) {
```
why do you write this spaces?

Reply:
didnt notice.done